src/base/utils/fs.cpp

```diff
@@ -313,7 +313,7 @@ bool Utils::Fs::isNetworkFileSystem(const QString &path)
     if (statfs(file.toLocal8Bit().constData(), &buf) != 0)
         return false;
 
-#ifdef Q_OS_MAC
+#if defined Q_OS_MAC || defined Q_OS_OPENBSD
```

Comment:
Can you use 
```c++
#if defined(Q_OS_MAC) || defined(Q_OS_OPENBSD)
```

to keep the same style?

Reply:
Yes, sorry, amended.